docs/administration.md

```diff
@@ -150,7 +160,7 @@ in CoreMedia Studio and add a struct named
     Window to lie within the future in days.
     (_optional_, default: `0`, type:`Integer`, scope:**global**)
 * `retryCommunicationErrors` Number of retries in case of a communication error
-    with GlobalLink. (_optional_, default: `5`, type:`Integer`)
+    with GlobalLink or Content Management Server. (_optional_, default: `5`, type:`Integer`)
```

Comment:
isn't the number of retries for CMS communication errors unlimited, and not affected by this setting?